extensions/binding/org.eclipse.smarthome.binding.tradfri/README.md

```diff
@@ -26,6 +26,8 @@ The following matrix lists the capabilities (channels) for each of the supported
 
 The gateway requires a `host` parameter for the hostname or IP address and a `code`, which is the security code that is printed on the bottom of the gateway. Optionally, a `port` can be configured, but any standard gateway uses the default port 5684.
 
+The code is used during the initialization for retrieving unique username and pre-shared key from the gateway (fw version 1.2.42 onwards) and then it's discarded from the configuration. On each initialization if the code is present in the thing configuration - the unique username and pre-shared key are recreated and the code is again discarded.
```

Comment:
please add a line break after every sentence

Reply:
Note that "discarded" only works if the Thing is defined in a database and not in a file.

Reply:
You should also mention the two newly introduced config parameters and explain how they can be used (as an alternative to the security code).

Reply:
>Note that "discarded" only works if the Thing is defined in a database and not in a file.

How is then (if at all) configuration change persisted in a file ? Is it in general possible to alter a thing defined in a file ?

If not - I suppose we can't fulfill the requirement to not store the security code and use the identity instead

Reply:
No, files cannot be altered.

> If not - I suppose we can't fulfill the requirement to not store the security code and use the identity instead

WE are not storing the code, it is the user. And once he has successfully authenticated and sees the INFO logging with identity+psk, he can manually update his file to use this instead of the code (if he wants to avoid having the code stored in a file).